DMA clock is enabled by setting GCCR bit 3 instead of 4

Signed-off-by: Gwenhael Goavec-Merou <gwenhael.goavec-merou@armadeus.com>
---

Index: linux-3.8-rc1/arch/arm/mach-imx/clk-imx1.c
===================================================================
--- linux-3.8-rc1.orig/arch/arm/mach-imx/clk-imx1.c	2012-12-22 02:19:00.000000000 +0100
+++ linux-3.8-rc1/arch/arm/mach-imx/clk-imx1.c	2013-01-01 21:19:56.000000000 +0100
@@ -72,7 +72,7 @@
 	clk[per3] = imx_clk_divider("per3", "spll", CCM_PCDR, 16, 7);
 	clk[clko] = imx_clk_mux("clko", CCM_CSCR, 29, 3, clko_sel_clks,
 			ARRAY_SIZE(clko_sel_clks));
-	clk[dma_gate] = imx_clk_gate("dma_gate", "hclk", SCM_GCCR, 4);
+	clk[dma_gate] = imx_clk_gate("dma_gate", "hclk", SCM_GCCR, 3);
 	clk[csi_gate] = imx_clk_gate("csi_gate", "hclk", SCM_GCCR, 2);
 	clk[mma_gate] = imx_clk_gate("mma_gate", "hclk", SCM_GCCR, 1);
 	clk[usbd_gate] = imx_clk_gate("usbd_gate", "clk48m", SCM_GCCR, 0);
